src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRecordingController.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.darts.audio.controller;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.darts.audio.service.AudioService;
+import uk.gov.hmcts.darts.audiorecording.api.AudioRecordingApi;
+import uk.gov.hmcts.darts.audiorecording.model.AddAudioRequest;
+
+@RestController
+@RequiredArgsConstructor
+public class AudioRecordingController implements AudioRecordingApi {
```

Comment:
Please check with @hemantasharma1129 - but I have a feeling that this new endpoint should exist under the `AudioController implements AudioApi`. We have some tech debt issues around the paths and operations that exist in the Audio API - some need to be moved at some point to the Audio Requests API.

I think we intend to have the endpoint operations Path starting with `/audios` for the `media` operations - so yours will be the first POST endpoint.

Reply:
The endpoint should be under AudioController.

Reply:
ACCEPT.  I wasn't sure because the audio controller had a lot of different endpoint paths.